webpack.config.js

```diff
@@ -0,0 +1,66 @@
+const path = require("path");
+
+const commonConfig = {
+  entry: "./src/index.ts",
+  devtool: "source-map",
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        use: "ts-loader",
+        exclude: /node_modules/
+      }
+    ]
+  },
+  resolve: {
+    extensions: [".tsx", ".ts", ".js"]
+  }
+};
+
+module.exports = [
+  {
+    ...commonConfig,
```

Comment:
This works here because the two objects have distinct sets of keys, but consider switching to https://github.com/survivejs/webpack-merge for a more robust webpack config merging solution.

Reply:
Will definitely keep that in mind in case the keys ever become non-distinct.